### PR TITLE
Support markbind build/include/render/serve from another directory

### PR DESCRIPTION
**Pull request by rachx**
See https://github.com/MarkBind/markbind/pull/179#discussion_r183778221

### CODE REVIEW

**Comment by acjh** [comment]
This doesn't work yet, right?

File: lib/Site.js [hunk 1]
```diff
@@ -66,7 +67,8 @@ const USER_VARIABLES_DEFAULT = '<span id="example">\n'
 const GENERATE_SITE_LOGGING_KEY = 'Generate Site';
 
 function Site(rootPath, outputPath) {
-  this.rootPath = rootPath;
+  const configPath = findUp.sync(SITE_CONFIG_NAME, { cwd: rootPath });
+  this.rootPath = (configPath && path.dirname(configPath)) || rootPath;
```
Comment (acjh) [inline lib/Site.js]: A ternary operator is clearer:

```js
this.rootPath = configPath ? path.dirname(configPath) : rootPath;
```

**Comment by rachx** [comment]
How did you get the error?
It works locally for me and there was no difference in test_site/index.html.
The filename does not contain my desktop folder name but it does has a backslash \ though. A unix/mac thing? `/test_site\requirements\SpecifyingRequirements._include_.html`

**Comment by acjh** [comment]
I ran `markbind serve test/test_site` from the root directory of MarkBind.

**Pull request merged**